static/js/containers/test_util.js

```diff
@@ -26,7 +26,7 @@ export const SUCCESS_ACTIONS: Array<ActionType> = [
   RECEIVE_GET_USER_PROFILE_SUCCESS,
 ];
 
-export const DASHBOARD_SUCCESS_ACTIONS = SUCCESS_ACTIONS.concat([
+export const SUBSET_DASHBOARD_SUCCESS_ACTIONS = SUCCESS_ACTIONS.concat([
```

Comment:
`SUBSET` is a little less than clear, could we call this `DASHBOARD_SUCCESS_NO_LEARNERS_ACTIONS` or something similar?